app/views/booking_requests/index.html.erb

```diff
@@ -9,9 +9,11 @@
 
 <div class="row">
   <div class="col-md-12">
-    <% if @booking_requests.empty? %>
+    <% if @booking_requests.all.empty? %>
       <p class="no-content t-notice">No booking requests requiring attention.</p>
     <% else %>
+      <%= paginate @booking_requests.entities %>
```

Comment:
I agree. This `LocationAwareEntities` decorator is used for displaying appointments too. I'm going to get those paginated then circle back around and tidy up this naming.